stories/Input/SelectStory.tsx

```diff
@@ -26,9 +26,28 @@ const props = {
       name: 'status',
       type: 'string',
       defaultValue: '',
-      possibleValue: <code>success | error</code>,
+      possibleValue: <code>error</code>,
       require: 'no',
-      description: 'Sets different style for Select based on status.',
+      description:
+        '(DEPRECATED) Sets different style for Select based on status.',
+    },
+    {
+      name: 'error',
+      type: 'boolean | string',
+      defaultValue: '',
+      possibleValue: 'any',
+      require: 'no',
+      description:
+        'Sets error state on component. If string, the error will be shown below the select',
+    },
+    {
+      name: 'ErrorComponent',
+      type: 'Component',
+      defaultValue: '<DefaultError />',
```

Comment:
I think we can just show that it is a `<span color="#hexcode" />`

Reply:
Updated!